validator/remote/src/main/java/tech/pegasys/teku/validator/remote/apiclient/OkHttpValidatorRestApiClient.java

```diff
@@ -166,8 +173,8 @@ public OkHttpValidatorRestApiClient(final HttpUrl baseEndpoint, final OkHttpClie
     queryParams.put("randao_reveal", encodeQueryParam(randaoReveal));
     graffiti.ifPresent(bytes32 -> queryParams.put("graffiti", encodeQueryParam(bytes32)));
 
-    if (blinded) {
-      throw new UnsupportedOperationException("blinded blocks not yet implemented");
+    if (blindedBlocksEnabled && blinded) {
```

Comment:
Shouldn't this be `||` instead of `&&`? ie if you explicitly request a blinded block we should request a blinded block but also if we're always using the blinded block creation API (blindedBlocksEnabled) then we should

Reply:
Actually it's possible this is the wrong level to do the feature toggle at.  It might be more intuitive to have ValidatorRestApiClient just expose separate `createUnsignedBlock` and `createUnsignedBlindedBlock` methods (matching the API) and then `RemoteValidatorApiHandler` would have the feature toggle and select between the blinded and unblinded apis.

Reply:
so this is more 'if we're using blinded blocks then I care, otherwise i don't'...
that's why i've gone with `&&`, because then the CLI will disable the api option basically...
the blinded option was the way it was handling things but the CLI should disable that, so unless i rework things...

Reply:
my theory was that's what makes it a 'feature toggle' - it toggles things off if its not set...